model/src/main/java/com/javahelp/model/token/Token.java

```diff
@@ -0,0 +1,123 @@
+package com.javahelp.model.token;
+
+import com.javahelp.model.user.User;
+import org.joda.time.DateTime;
+import java.security.SecureRandom;
+
+/**
+ * Representation of token entity
+ */
+
+public class Token {
+    /**
+     * Token requested by the user
+     */
+    private String token;
+
+    /**
+     * Unique id of the token.
+     */
+    private String id;
+
+    /**
+     * Expiry date of the token.
+     */
+    private DateTime expiry;
+
+    /**
+     * Issued date of the token.
+     */
+    private DateTime issued;
+
+    /**
+     * User who requested the token.
+     */
+    private User user;
+
+    /**
+     * This holds any additional information about the toekn.
+     */
+    private String tag;
+
+
+    /**
+     * Creates a new {@link Token}
+     * @param tag {@link String} of this {@link Token}
+     * @param user {@link User} who requested this {@link Token}
+     */
+    public Token(String tag, User user){
+        // Creating a token using functions from SecureRandom
+
+        SecureRandom random = new SecureRandom();
+        byte bytes[] = new byte[20];
```

Comment:
A small comment on Java notation, is this like in ``C`` syntax? 

Because usually I see in Java, we do ``byte[] bytes = new byte[20];`` instead.

Reply:
Ye you can put it after either the type or the name, but it's probably more conventional to put it after the type